ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/ldes/eventstream/services/EventStreamConverterImpl.java

```diff
@@ -55,9 +52,17 @@ private List<Statement> addViewStatements(List<TreeNode> views) {
 
 		views.forEach(view -> {
 			Resource viewResource = createResource(view.getFragmentId());
-
 			statements.add(createStatement(viewResource, RDF_SYNTAX_TYPE, createResource(TREE_NODE_RESOURCE)));
-			statements.addAll(relationStatementConverter.getRelationStatements(view.getRelations(), viewResource));
+			view.getRelations()
+					.forEach(treeRelation -> {
+						TreeRelationResponse treeRelationResponse = new TreeRelationResponse(treeRelation.treePath(),
+								treeRelation.treeNode(),
+								ldesConfig.getHostName() + "/" + ldesConfig.getCollectionName()
```

Comment:
I noticed that we do this exact same concatenation on 7 different places. Perhaps we can add a getter that does this for us in the LdesConfig?

Reply:
Top comment! Implemented!